packages/v/volk/xmake.lua

```diff
@@ -1,23 +1,25 @@
 package("volk")
 
-    set_homepage("https://github.com/zeux/volk")
-    set_description("volk is a meta-loader for Vulkan")
-    set_license("MIT")
+set_homepage("https://github.com/zeux/volk")
+set_description("volk is a meta-loader for Vulkan")
+set_license("MIT")
 
-    add_urls("https://github.com/zeux/volk/archive/$(version).tar.gz",
-             "https://github.com/zeux/volk.git")
-    add_versions("1.2.190", "07f03720b8c70a626c98cc9545350538122bca9f853e6ed20ccad5a25d55fa4b")
-    add_versions("1.2.162", "ac4d9d6e88dee5a83ad176e2da57f1989ca2c6df155a0aeb5e18e9471aa4d777")
+add_urls("https://github.com/zeux/volk/archive/$(version).tar.gz",
+         "https://github.com/zeux/volk.git")
+add_versions("1.2.190", "07f03720b8c70a626c98cc9545350538122bca9f853e6ed20ccad5a25d55fa4b")
+add_versions("1.2.162", "ac4d9d6e88dee5a83ad176e2da57f1989ca2c6df155a0aeb5e18e9471aa4d777")
+add_versions("1.3.204", "7776e7f3c70f199579da33d2ccd7152ca8b96182fa98c31fbe80880cef0fdf70")
 
-    add_deps("vulkan-headers")
+add_deps("vulkan-headers")
 
-    add_configs("shared", {description = "Build shared library.", default = false, type = "boolean", readonly = true})
```

Comment:
please do not remove it, it is only static or headeronly library. we need disable shared config and mark it as readonly.